### ISSUE TITLE
training details

### ISSUE THREAD

**Issue by kingyj7** [aurooj/Hand-Segmentation-in-the-Wild#6]
Hi, thank you to share your great work! 
    Now I can finetune the refinenet on egohand dataset, but to achieve the best result of miou=81.4 in your paper, how to set the hyperparameters? Can you share your training strategy? Thank you in advance.

**Comment by aurooj** [comment]
@kingyj7 Thanks for your interest in this work.
For
training RefineNet, we used a learning rate of 5e-5 with an increased learning rate by a factor of 10 for our binary classification layer. All models were trained till convergence, and multi-scale evaluation is used with three scales: [0.6, 0.8 and 1.0].
We fine-tuned RefineNet-Res101 on EgoHands dataset for 80 epochs.

**Comment by kingyj7** [comment]
Hi, @aurooj 
Thanks for your quick reply! But I still have several following questions.
1. In fact, I'm confused by 
> a learning rate of 5e-5 with an increased learning rate by a factor of 10   

Could you please explain how the learning rate would change during 80 epochs?

2. I'm also wondering if the crop_size of input matters since the image resolution in EgoHands with 1280*720 is larger than pascal-person-part. Concretely, how did you set the following parameters in demo_refinenet_train.m
```
% control the size of input images  
run_config.input_img_short_edge_min=450;  
run_config.input_img_short_edge_max=1100;

% random crop training:
run_config.crop_box_size=400;
```
Looking forward your answers.
Best regards,

**Comment by aurooj** [comment]
Learning rate didn't change during training if I correctly remember. 

> a learning rate of 5e-5 with an increased learning rate by a factor of 10

means that our network uses lr=5e-5 for all layers except the last layer which has lr=5e-4.

We used the same settings as were used in pascal person-part (given below): 
```
run_config.input_img_short_edge_min=450;
 run_config.input_img_short_edge_max=600;

% random crop training:
run_config.crop_box_size=400;
```

hope it helps!
